Update README.md (#473)
hashicorp/vscode-terraform: README.md

- Closes braces and quotes
- Includes `for_each` and `variable` syntax shortcuts (`fore`, `vare`, `varm`)

## Getting Started

> **IMPORTANT:** After installing, you must perform a `terraform init` to provide `terraform-ls` with an up-to-date Provider schema. The language-server will not work correctly without first completing this step!

Reviewer: ```suggestion
**IMPORTANT:** After installing, you must perform a `terraform init` to provide `terraform-ls` with an up-to-date provider schemas. The language server will not work correctly without first completing this step!
```
(minor nits) We never capitalize `provider` and technically there can be schemas for more than a single provider.

1. Install the extension from the Marketplace

> 1. Reload VS Code after the installation (click the reload button next to the extension).

Reviewer: Maybe too much context - unsure - but we could link to https://github.com/hashicorp/terraform-ls/issues/222 here - i.e. this step should not be necessary anymore once that issue is resolved

Author: I'd say we can leave out for these instructions, and just remove it from the steps once we enhance/add monitoring

> 1. Perform a `terraform init` to provide `terraform-ls` with an up-to-date Provider schema

Reviewer: ```suggestion
1. Perform a `terraform init` to provide `terraform-ls` with an up-to-date provider schemas
```
AFAIK we never capitalize `provider` anywhere.

> 1. Open your desired Workspace and/or the root folder containing your Terraform files. Note: see [*Known Issues*](#known-issues) below about multi-folder workspaces

Reviewer: ```suggestion
1. Open your desired workspace and/or the root folder containing your Terraform files. Note: see [*Known Issues*](#known-issues) below about multi-folder workspaces
```

I may not be fully familiar with the VS Code naming convention here, but is "workspace" the official name for it? Either way I think it probably shouldn't be capitalized.

> 1. Depending on your settings in VS Code, completion will start automatically (if not inside quotes/string literal, on certain trigger characters), or you can explicitly trigger completion via keyboard combination (Ctrl+Space on Windows, control+space on Mac).

Reviewer: Similar to above we could link to https://github.com/hashicorp/vscode-terraform/issues/373 here, but I'm unsure if the context is helpful.

> 1. Depending on your settings in VS Code, completion will start automatically (if not inside quotes/string literal, on certain trigger characters), or you can explicitly trigger completion via keyboard combination (Ctrl+Space on Windows, control+space on Mac).

Reviewer: ```suggestion
1. Depending on your settings in VS Code, completion will start automatically (if not inside quotes/string literal, on certain trigger characters), or you can explicitly trigger completion via keyboard combination (`Ctrl+Space` by default).
```
